athenet/layers/fully_connected.py

```diff
@@ -63,7 +63,9 @@ def output_shape(self):
     def _reshape_input(self, raw_layer_input):
         """Return input in the correct format for fully connected layer.
 
-        :raw_layer_input: Input in the format (n_in, n_out) or compatible.
+        :param raw_layer_input: Input in the format (n_batches, n_channels) or
```

Comment:
Fully connected layer does not have notion of channels. It should be (n_batches, n_in), or more specifically, (n_batches, n_input_neurons)